prime-router/src/test/kotlin/FileSettingsTests.kt

```diff
@@ -64,21 +66,18 @@ class FileSettingsTests {
 
     @Test
     fun `test loading a receiver`() {
-        val settings = FileSettings()
-        settings.loadOrganizations(ByteArrayInputStream(receiversYaml.toByteArray()))
+        val settings = FileSettings().also { it.loadOrganizations(ByteArrayInputStream(receiversYaml.toByteArray())) }
         val result = settings.findReceiver("phd1.elr")
-
-        assertEquals(1, result?.jurisdictionalFilter?.size)
+        assertThat(result?.jurisdictionalFilter).isNotNull().hasSize(1)
     }
 
     @Test
     fun `test loading a sender and receiver`() {
-        val settings = FileSettings()
-        settings.loadOrganizations(ByteArrayInputStream(sendersAndReceiversYaml.toByteArray()))
-
+        val settings = FileSettings().also {
+            it.loadOrganizations(ByteArrayInputStream(sendersAndReceiversYaml.toByteArray()))
+        }
```

Comment:
Kotlin has this fun `.also` scope function that lets act on the variable you're creating and then returns the object you're acting on.